Godex does not compile with float=64

It's a new master branch feature.
error given:
```
godex\modules\bullet_physics\overlap_check.cpp(27): error C2664: 'void btConvexShape::project(const btTransform &,const btVector3 &,btScalar &,btScalar &,btVector3 &,btVector3 &) const': cannot convert argument 3 from 'real_t' to 'btScalar &'
godot\thirdparty\bullet\BulletCollision\CollisionShapes\btConvexShape.h(53): note: see declaration of 'btConvexShape::project'
godex\modules\bullet_physics\overlap_check.cpp(758): error C2259: 'overlap_check_polyhedron_polyhedron::NoRes': cannot instantiate abstract class
godex\modules\bullet_physics\overlap_check.cpp(754): note: see declaration of 'overlap_check_polyhedron_polyhedron::NoRes'
godex\modules\bullet_physics\overlap_check.cpp(758): note: due to following members:
godex\modules\bullet_physics\overlap_check.cpp(758): note: 'void btDiscreteCollisionDetectorInterface::Result::addContactPoint(const btVector3 &,const btVector3 &,btScalar)': is abstract
godot\thirdparty\bullet\BulletCollision\NarrowPhaseCollision\btDiscreteCollisionDetectorInterface.h(36): note: see declaration of 'btDiscreteCollisionDetectorInterface::Result::addContactPoint'
systems_base.cpp
```

Still having problems after adding those two lines in godex scons:
```
[Initial build] godex\modules\bullet_physics\overlap_check.cpp(27): error C2664: 'void btConvexShape::project(const btTransform &,const btVector3 &,btScalar &,btScalar &,btVector3 &,btVector3 &) const': cannot convert argument 3 from 'real_t' to 'btScalar &'
[Initial build] godot\thirdparty\bullet\BulletCollision\CollisionShapes\btConvexShape.h(53): note: see declaration of 'btConvexShape::project'Compiling ==> godex\modules\bullet_physics\systems_walk.cpp

systems_base.cpp
godex\modules\bullet_physics\overlap_check.cpp(758): error C2259: 'overlap_check_polyhedron_polyhedron::NoRes': cannot instantiate abstract class
godex\modules\bullet_physics\overlap_check.cpp(754): note: see declaration of 'overlap_check_polyhedron_polyhedron::NoRes'systems_walk.cpp

godex\modules\bullet_physics\overlap_check.cpp(758): note: due to following members:
godex\modules\bullet_physics\overlap_check.cpp(758): note: 'void btDiscreteCollisionDetectorInterface::Result::addContactPoint(const btVector3 &,const btVector3 &,btScalar)': is abstract
godot\thirdparty\bullet\BulletCollision\NarrowPhaseCollision\btDiscreteCollisionDetectorInterface.h(36): note: see declaration of 'btDiscreteCollisionDetectorInterface::Result::addContactPoint'

```

--- a/modules/bullet_physics/SCsub
+++ b/modules/bullet_physics/SCsub
@@ -206,6 +206,9 @@ if env["builtin_bullet"]:
     if env["target"] == "debug" or env["target"] == "release_debug":
         env_bullet.Append(CPPDEFINES=["DEBUG"])
 
+    if env["float"] == "64":
+        env_bullet.Append(CPPDEFINES=["BT_USE_DOUBLE_PRECISION"])
+
     env_bullet.Append(CPPDEFINES=["BT_USE_OLD_DAMPING_METHOD"])
 
     env_thirdparty = env_bullet.Clone()

